stdlib/uri/0/mailto.rbs

```diff
@@ -4,5 +4,89 @@ module URI
   # RFC6068, the mailto URL scheme.
   #
   class MailTo < Generic
+    EMAIL_REGEXP: Regexp
+
+    # <!--
+    #   rdoc-file=lib/uri/mailto.rb
+    #   - build(args)
+    # -->
+    # ## Description
+    #
+    # Creates a new URI::MailTo object from components, with syntax checking.
+    #
+    # Components can be provided as an Array or Hash. If an Array is used, the
+    # components must be supplied as `[to, headers]`.
+    #
+    # If a Hash is used, the keys are the component names preceded by colons.
+    #
+    # The headers can be supplied as a pre-encoded string, such as
+    # `"subject=subscribe&cc=address"`, or as an Array of Arrays like `[['subject',
+    # 'subscribe'], ['cc', 'address']]`.
+    #
+    # Examples:
+    #
+    #     require 'uri'
+    #
+    #     m1 = URI::MailTo.build(['joe@example.com', 'subject=Ruby'])
+    #     m1.to_s  # => "mailto:joe@example.com?subject=Ruby"
+    #
+    #     m2 = URI::MailTo.build(['john@example.com', [['Subject', 'Ruby'], ['Cc', 'jack@example.com']]])
+    #     m2.to_s  # => "mailto:john@example.com?Subject=Ruby&Cc=jack@example.com"
+    #
+    #     m3 = URI::MailTo.build({:to => 'listman@example.com', :headers => [['subject', 'subscribe']]})
+    #     m3.to_s  # => "mailto:listman@example.com?subject=subscribe"
+    #
+    def self.build: (Array[String]) -> instance
```

Comment:
The official document says `URI::MailTo.build` takes a tuple of strings. So it would be correct and accurate to use `([String, String]) -> instance` instead of this line. But it's not good for use because Steep recognizes the type of tuple literal of strings as an `Array[String]`.